apps/pharos_web/web/core/real_search.ex

```diff
@@ -3,19 +3,40 @@ defmodule Core.RealSearch do
   def execute(%{"topic" => topic, "amount" => amount}) do
     query = %{topic: topic, amount: String.to_integer(amount)}
 
```

Comment:
Yeah, I'd say we keep this on our side. We could maybe have a separate branch locally that we checkout when we want to use that code if need be. We can show them the progression on the monitor or something. 

In terms of tests we can have a couple of tests using the fake approach we've done before. In fact the fake will only be necessary when we switch to a distributed version because with the regular services we can just call directly to them. We can still apply the fake at that point though to avoid the external request to each service (twitter etc).